src/sidebar/build-thread.js

```diff
@@ -165,10 +159,11 @@ function threadAnnotations(annotations) {
  *
  * @param {Thread} thread
  * @param {(Thread) => Thread} mapFn
```

Comment:
```suggestion
 * @param {(t: Thread) => Thread} mapFn
```

This is a mistake in the previous code, but `(Thread) => Thread` is interpreted by TS as a function that takes an argument of _any_ type _called_ `Thread` and returns a `Thread`.